### ISSUE TITLE
[Safari] In half-sized window, modals push content wider and nav bar expands

### ISSUE THREAD

**Issue by jinluyuan** [CenterForOpenScience/osf.io#3436]
### Steps
1. Go to osf.io on Safari
2. Make your window take up half the screen (I'm using a 27" Thunderbolt display), or small enough so the user's name is not displayed, only user's the Gravatar 
3. Go to the settings page of any one of your projects
4. Check off the box of one of the add-ons to select it
5. Notice the modal that pops up
6. Click "Cancel" in the modal
7. Click "Deauthorize" for any add-on
8. Notice the modal that pops up
9. Click "Cancel" in the modal
10. Do the same for the user settings Configure Add-ons

### Expected behavior
I expected that nothing in the background would move when the modals pop up.

### Actual behavior
The name of my OSF account, "Purple Dinosaur", appears when the modal appears, expanding the nav bar. The content of the page also expands. This may be related to https://github.com/CenterForOpenScience/osf.io/issues/1663, but this does not occur on the latest versions of Chrome, Firefox, or IE.

![movingmodulebug2](https://cloud.githubusercontent.com/assets/8591162/8484232/7eb295fe-20c4-11e5-956e-933561468a5e.gif)

**Comment by caneruguz** [comment]
@jinluyuan this issue has been discussed before: https://github.com/CenterForOpenScience/osf.io/issues/1663

I recommend closing. @sloria @lbanner thoughts?

**Comment by lbanner** [comment]
OK -- yes, close in favor of discussion and approach in #1663

**Comment by jinluyuan** [comment]
@caneruguz @lbanner This is not the same issue as #1663, though it may be related. This issue only applies to Safari, and the issue is that the content of the page expands.

**Comment by caneruguz** [comment]
@jinluyuan the content of the page expands because modals remove scrolling by adding a class to the body. When scrolling is removed, the scroll bar is removed, thus giving the page about 15 more pixels. Normally this shouldn't make the page wider but bootstrap has break points for responsive rules. In your case above the screen grab was taken at a point where the difference of 15-20 pixels switched the view from one size to another hence changing the view dramatically. 

I am also confident that this is not a Safari issue but one couldn't replicate it in other browser because one  would need to be at about the exact width for this switch to occur.

**Comment by jinluyuan** [comment]
@caneruguz This happens on Safari when the user makes the window progressively smaller, right until the user's name disappears from the top nav bar and only the Gravatar is visible. Then opening up a modal causes the content to expand - most notably, the user's name appears once again in the top nav bar. This is due to the scrollbar disappearing, as you say. The window has to be at the right size for this to happen, within a range of about 10 pixels (I measured it), which makes sense given your explanation about the breaking point. This window size is very common for me to use since it is about half the size of the screen on a 27" display.

However, on Chrome and Firefox, this behavior never occurs. The nav bar shifts to the right a little, as reported in #1663, but no matter how close I get to the appropriate window size (where the nav bar changes to only displaying the Gravatar), I cannot replicate the same effect of the content expanding.

This is obviously a minor issue, but I just wonder if there could be a way to make site behave on Safari the same way it does on Chrome & Firefox. Sorry that this is so nit-picky!

I closed my other issue (#2947) that was actually an exact repeat of #1663. I apologize for that one being a repeat.
